### PR TITLE
test: update accessibility tests

### PR DESCRIPTION
**Pull request by DlgSHi**
### Proposed behaviour

- update cypress version
- update cucumber preprocessor version
- add more CI machines for axe run
- fix for integrationFolder

### Current behaviour

We need to:
- update cypress version
- update cucumber preprocessor version
- add more CI machines for axe run
- fix for integrationFolder

### Checklist

- [x] Commits follow our style guide
- [ ] Related issues linked in commit messages if required
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required
- [ ] Typescript `d.ts` file added or updated if required

#### QA

- [ ] Tested in CodeSandbox/storybook
- [x] Add new Cypress test coverage if required
- [ ] Carbon implementation matches Design System/designs
- [x] UI Tests GitHub check reviewed if required

### CODE REVIEW

**Review by sianford — APPROVED** [review]

**Review by sianford — DISMISSED** [review]

File: cypress/README.md [hunk 1]
```diff
@@ -43,13 +58,13 @@ Storybook must be built and running before cypress can run.
...
-1. `npm start` - runs Storybook.
-2. `wait-on http://localhost:9001` - waits until Storybook is up and running and is ready to run tests.
+1. `npm start` - runs Storybook (regular tests) or run `npx http-server storybook-static -p 9001` to serve `static` folder as a Storybook server for `accessibility` tests (we already had built the static folder so no need to `start` storybook separatelly).
+2. `wait-on http://localhost:9001` - waits until Storybook is up and running and is ready to run tests (`CI` only).
 
 ### GitHub Actions
 1. `cypress.yml`
-- `npx cypress run --browser chrome --parallel -–record --spec './cypress/features/regression/**/*.feature'` - runs the complete test suite (except `accessibility`).
-- `npx cypress run --browser chrome --parallel -–record --spec './cypress/accessibility/*.test.js'` - runs the `accessibility` test suite only.
+- `npx cypress run --browser chrome --parallel -–record --spec './cypress/integration/regression/**/*.feature'` - runs the complete test suite (except `accessibility`).
```
Comment (samtjo) [inline cypress/README.md:66]:
```suggestion
- `npx cypress run --browser chrome --parallel -–record --spec './cypress/integration/**/*.feature'` - runs the complete test suite (except `accessibility`).
```

File: cypress/README.md [hunk 2]
```diff
@@ -25,7 +40,7 @@ Storybook must be running before Cypress tests can be run:
 1. Run Storybook `npm start`.
 2. Open a new terminal in the root path of the project.
 3. Run Cypress using the runner with `npx cypress open` or `npm run test-cypress`, then select the required feature file. Test results can be seen directly in the Cypress Test Runner UI.
-4. To run specific Cypress tests at the command line (headless browser for continuous integration) use: `npx cypress run --spec 'cypress/features/[tests-type]/[featureFileName].feature'`. Test results can be seen in the console run summary.
+4. To run specific Cypress tests at the command line (headless browser for continuous integration) use: `npx cypress run --spec 'cypress/integration/[tests-type]/[featureFileName].feature'|.test.js`. Test results can be seen in the console run summary.
```
Comment (samtjo) [inline cypress/README.md:43]:
```suggestion
4. To run specific Cypress tests at the command line (headless browser for continuous integration) use: `npx cypress run --spec 'cypress/integration/[tests-type]/[featureFileName](.feature|.test.js)'`. Test results can be seen in the console run summary.
```

Might not be exactly the right syntax but should be something like that I think

Comment (DlgSHi) [reply cypress/README.md:43]: it is not a syntax just to show what we can run :)

Comment (DlgSHi) [reply cypress/README.md]: updated

File: cypress/README.md (hunk 1 above)
Comment (DlgSHi) [reply cypress/README.md]: updated

**Review by samtjo — DISMISSED** [review]

**Review by sianford — APPROVED** [review]

**Review by samtjo — APPROVED** [review]

**Pull request merged**